smart_campus/app/templates/app/index.html

```diff
@@ -1,4 +1,47 @@
-{% extends "app/base.html" %}
-{% load static %}
-
-{% block title %}Smart Campus{% endblock %}
+{% extends 'app/base.html' %} {% load static %} {% block stylesheet %} {{ block.super }}
+<style type="text/css">
+  .segment {
+    max-width: 300px;
+  }
+</style>
+{% endblock %} {% block content %}
+<div class="ui grid">
+  <div class="row">
+    <div class="four wide column centered">
+      <div class="ui small info message">
+        <i class="close icon"></i>
+        <div class="header">
+          改版資訊
+        </div>
+        <ul class="list">
+          <li>終於給index加了一些東西</li>
+          <li>讚讚讚</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="four wide column centered">
+      <div class="ui small info message">
+        <i class="close icon"></i>
+        <div class="header">
+          改版資訊
+        </div>
+        <ul class="list">
+          <li>終於給index加了一些東西</li>
+          <li>讚讚讚</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</div>
+{% endblock %} {% block script %} {{ block.super }}
```

Comment:
Add proper blank lines can enhance readability.
**DO NOT** remove them for no reason.

Reply:
okay, indented it.